scripts/feature_toggle_report_generator.py

```diff
@@ -291,6 +291,14 @@ def bool_for_null_numbers(n):
             else:
                 return False
 
+        def bool_for_null_lists(l):
+            if not l:
```

Comment:
Nit: could have `if l:` and reverse cases. Positives are easier on the brain. 🙂

Reply:
fixed.